@planetarium/account-aws-kms/tsconfig.json

```diff
@@ -1,15 +1,17 @@
 {
-  "include": ["./src", "*.d.ts", "*.ts"],
+  "include": ["./src", "*.d.ts", "src/*.ts"],
```

Comment:
Isn't it included by `./src`? 👀

Reply:
just src also works since it's "at" rootDir.